docs/code/IDataViewTypeSystem.md

```diff
@@ -383,82 +383,53 @@ Key types are instances of the sealed class `KeyType`, which derives from
 
 In addition to its underlying type, a key type specifies:
 
-* A count value, between `0` and `int.MaxValue`, inclusive
+* A count value, between `1` and `ulong.MaxValue`, inclusive
```

Comment:
At my first glance, I didn't know what does `inclusive` mean but then I realize how stupid I was after reading through the following paragraph. Maybe
```suggestion
* A count value, from `1` to `ulong.MaxValue`.
```
can help other naive users?

Reply:
I do not like the suggested change as much as I might like. Chaning "between" to "from" is good. (Between suggests exclusion.) Removing "inclusive" is bad. What you think?